portal-web/test/functional/com/liferay/portalweb/paths/portlet/ckeditorsample/CKEditorBalloonEditorSample.path

```diff
@@ -0,0 +1,63 @@
+<html>
+<head>
+<title>CKEditorBalloonEditorSample</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+	<tr><td rowspan="1" colspan="3">CKEditorBalloonEditorSample</td></tr>
+</thead>
+
+<tbody>
+
+<!--PARAGRAPH-->
+<tr>
+	<td>BODY_TEXT</td>
+	<td>//div[@class='portlet-body']/div/div/p[${paragraph_number}]</td>
```

Comment:
Let's avoid using non-semantic element tags in our path locators.

see for more tips if interested: https://docs.google.com/spreadsheets/d/1KgMk5CoAB0Kqylg2dufw9X4zGenrWNEv13KdVckrkvg/edit#gid=1536975616

Also, are we able to avoid using a paragraph number parameter?

Another thing I wanted to mention, `portlet-body` class doesn't seem specific enough for CKEditor. Maybe there's a more specific class that describes that BODY_TEXT element target better. ie: would CKEditor#BODY or CKEditor#BODY_TEXT work? It has `cke_editable` which would help to more clearly describe the element it's targeting. Your paths for the toolbar buttons are exactly what I'd be looking for ie: `cke_button__tablerow`. They really describe what the path is targeting.